docs/cargo_features.md

```diff
@@ -36,4 +36,6 @@
 |wayland|Enable this to use Wayland display server protocol other than X11.|
 |subpixel_glyph_atlas|Enable this to cache glyphs using subpixel accuracy. This increases texture memory usage as each position requires a separate sprite in the glyph atlas, but provide more accurate character spacing.|
 |bevy_ci_testing|Used for running examples in CI.|
+|tokio|Use tokio as task execution engine.|
```

Comment:
Are these two features mutually exclusive? If they are, this docs should cover that mutual exclusion.